crates/ubrn_bindgen/src/bindings/react_native/gen_typescript/templates/wrapper.ts

```diff
@@ -40,7 +40,8 @@ const {
 } = {{ entry.0.1 }}.converters;
 {%- endfor %}
 
-const uniffiIsDebug = (process?.env?.NODE_ENV !== "production" || {{ config.is_debug() }});
+// @ts-ignore -- The process global might not be defined
+const uniffiIsDebug = (typeof process !== "object" || typeof process.env !== "object" || process.env.NODE_ENV !== "production" || {{ config.is_debug() }});
```

Comment:
Does this mean if _either_ `process` or `process.env` are not an object, OR or `process.env.NODE_ENV` is not `"production"`, then `uniffiIsDebug` is set to `true`?

I'm not sure I understand the difference between this and the nullish coelecsing (`?.`) it replaces. If there is none,  I _think_ the intent of the `?.` version is more clear. What do you think?

Reply:
I've tried to match the semantics of the existing code.
I interpret this as: If both the `process` and `process.env` are objects the `NODE_ENV` must not be `"production"` for `uniffiIsDebug` to remain true.

> the difference between this and the nullish coelecsing (?.) it replaces

Using the `typeof` operator, you can provide an undefined identifier whereas the nullish coalescing operator expects the first variable to be (at least) defined in the scope - here's an example from a node REPL:

```
> missing?.env?.NODE_ENV === "production"
Uncaught ReferenceError: missing is not defined
> typeof missing === "object"
false
```

Reply:
Ah ok, I understand now. Thank you for being patient with me.

Reply:
Hmm - I just ran this locally and prettier is messing up the placement of the `// @ts-ignore` comment.
I'll push another commit to fix that.